http/auth-gateway.js

```diff
@@ -148,7 +148,7 @@ var HttpAuthGateway = HttpGateway.extend({
      */
     handleTokenExchangeSuccess : function(token, method, path, data, headers, resolve, reject, response)
     {
-        token = _.extend(token, response.data);
+        token = _.extend(token, response);
```

Comment:
Lively expects the response data to be in `response.data`.

Lively should override this method to perform its custom functionality.